### PR TITLE
Resolve linking errors for WD-07 publication

### PR DESCRIPTION
**Pull request by AngeloKai**
When you test, please make sure you have an updated copy of bikeshed. 

This resolves the two remaining linking errors: 
- AttestationNotPrivateError not found 
- [[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors) idl reference not found.

***
<a href="https://pr-preview.s3.amazonaws.com/AngeloKai/webauthn/pull/703.html" title="Last updated on Dec 5, 2017, 7:30 AM GMT (4255136)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/703/2f0b13e...AngeloKai:4255136.html" title="Last updated on Dec 5, 2017, 7:30 AM GMT (4255136)">Diff</a>

### CODE REVIEW

**Comment by AngeloKai** [comment]
Can one of the reviewers I nominated please double check in case the change is not right? Anyone reviewing it would be fine.

**Comment by jcjones** [comment]
I agree with removing the link for `AttestationNotPrivateError`, as it's our own.

The `[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)` is indeed fixed.

I still show one linking error with an updated bikeshed, checking on it:
```
LINK ERROR: No 'idl' refs found for 'id' with for='PublicKeyCredentialEntity'.
<a data-link-type="idl" data-lt="id" data-link-for="PublicKeyCredentialEntity" data-link-for-hint="PublicKeyCredential/[[Create]](origin, options, sameOriginWithAncestors)">id</a>
```

File: index.bs [hunk 1]
```diff
@@ -986,7 +986,8 @@ for="PublicKeyCredential" method>\[[CollectFromCredentialStore]](origin, options
 
 <div link-for-hint="PublicKeyCredential/[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)">
 
-This [=internal method=] accepts three arguments:
+{{PublicKeyCredential}}'s [=interface object=]'s implementation of the 
+<dfn for="PublicKeyCredential" method>\[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)</dfn> [=internal method=] accepts three arguments:
```
Comment (equalsJeffH) [inline index.bs]: let's please not put this long unnecessary sentence back and find another place for the \<dfn>.  apologies for not cleaning this up myself earlier...  am looking into it....

**Comment by equalsJeffH** [comment]
@AngeloKai -- thx for working on fixing these linking errors, my apologies for not having done so -- the "AttestationNotPrivateError" one is fine, the other change I'd rather not make (https://github.com/w3c/webauthn/pull/703/files#r154392024) -- am looking into a workaround...

**Comment by equalsJeffH** [comment]
@jcjones -- wrt https://github.com/w3c/webauthn/pull/703#issuecomment-348516051 -- fwiw, i do not get any errors with the present state of the anglokai/webauthn:master branch when building using cloud bikeshed.

**Comment by equalsJeffH** [comment]
Ok, I have an edit on the `[[DiscoverFromExternalSource]]` \<dfn> thing  that builds w/o errors -- is it ok for me to push it to anglokai/webauthn:master branch ?

**Comment by equalsJeffH** [comment]
@AngeloKai sez that my pushing directly to https://github.com/AngeloKai/webauthn:master at this time is inconvenient, so here's a hist of the latest commits in that branch including mine in my local clone: 
```
 > githist
* 26af148 2017-12-01 | edit [[DiscoverFromExternalSource]] dfn & markup (HEAD -> master) [JeffH]
* 6ac3d01 2017-12-01 | remove unnecessary commits (origin/master, origin/HEAD) [Angelo Liao]
* 2c936a6 2017-12-01 | resolve linking error: idl ref not found for [[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors) [Angelo Liao]
* e04880f 2017-12-01 | resolve linking error for AttestationNotPrivateError [Angelo Liao]
*   234a61f 2017-12-01 | Merge branch 'master' of https://github.com/w3c/webauthn [Angelo Liao]
```
and here's the diff of what I propose we change, between the two most-recent commits shown above:
```
> git diff 6ac3d01..26af148
diff --git a/index.bs b/index.bs
index 2efe8d1..29792f3 100644
--- a/index.bs
+++ b/index.bs
@@ -982,12 +982,11 @@ for="PublicKeyCredential" method>\[[CollectFromCredentialStore]](origin, options
 {{Credential/[[CollectFromCredentialStore]]()|Credential.[[CollectFromCredentialStore]]()}}, of returning an empty set.
 
 
-<h5 id="discover-from-external-source" algorithm>PublicKeyCredential's `[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)` method</h5>
+<h5 id="discover-from-external-source" algorithm>PublicKeyCredential's <code><dfn for="PublicKeyCredential" method>\[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)</dfn></code> method</h5>
 
 <div link-for-hint="PublicKeyCredential/[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)">
 
-{{PublicKeyCredential}}'s [=interface object=]'s implementation of the 
-<dfn for="PublicKeyCredential" method>\[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)</dfn> [=internal method=] accepts three arguments:
+This [=internal method=] accepts three arguments:
 
 <dl dfn-type="argument" dfn-for="PublicKeyCredential/[[DiscoverFromExternalSource]](origin, options, sameOriginWithAncestors)">
 
```

**Comment by equalsJeffH** [comment]
shall we close this PR ?

**Comment by equalsJeffH** [comment]
doh, nevermind.  it's merged already, had neglected to refresh the page

**Pull request merged**